pandas/tests/series/test_constructors.py

```diff
@@ -1810,6 +1810,20 @@ def test_constructor_bool_dtype_missing_values(self):
         expected = Series(True, index=[0], dtype="bool")
         tm.assert_series_equal(result, expected)
 
+    def test_constructor_int64_dtype(self, any_int_dtype):
+        # GH-44923
+        result = Series(["0", "1", "2"], dtype=any_int_dtype)
+        expected = Series([0, 1, 2], dtype=any_int_dtype)
+        tm.assert_series_equal(result, expected)
+
+    def test_constructor_float64_dtype(self, any_float_dtype):
+        # GH-44923
+        if any_float_dtype in ["Float32", "Float64"]:
```

Comment:
is there an issue for this?

Reply:
Float -> Float32, Float64 are failing as it is unable to implicitly cast strings. As the implicit cast fails so I am xfailing them, 

#### ```u can xfail the failing ones that are tricky to fix eg Float but the others should work```

This one you had mentioned above.

Reply:
i think these now work in main #45424